recipe/meta.yaml

```
@@ -1,39 +1,91 @@
 {% set name = "evidently" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.39" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/evidentlyai/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 49f3f5a724e65369f33243732dacd48aad68e3aa0bd66e3a8e7784ed5f675b5f
+  sha256: c07a65a5c07138708328dbdf1e2b369f725039f714efb393b5631db7b57fa393
+  patches:
+    - 0001-patch-test-to-exclude-experimental-features.patch
 
 build:
   number: 0
   # umap-learn isn't available on s390x
   skip: True  # [py<37 or s390x]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # missing dynaconf for py312
+  skip: True  # [py>311]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
     - python
     - setuptools
     - wheel
   run:
-    - nltk >=3.6.7
-    - numpy >=1.19.5
-    - pandas >=1.3.5
-    - plotly >=5.5.0
-    - pydantic >=1.9,<2
     - python
-    - pyyaml >=5.1
-    - requests >=2.19.0
-    - scikit-learn >=0.24.0
-    - scipy >=1.5.4
+    - plotly >=5.10.0
     - statsmodels >=0.12.2
-    - umap-learn >=0.5.3
+    - scikit-learn >=1.0.1
+    - pandas >=1.3.5
+    - numpy >=1.22.0,<2.1
+    - nltk >=3.6.7
+    - scipy >=1.10.0
+    - requests >=2.32.0
+    - PyYAML >=5.4
+    - pydantic >=1.10.13
+    - litestar >=2.8.3
+    - typing_inspect >=0.9.0
+    - uvicorn >=0.22.0
```

It's asking for standard here: https://github.com/evidentlyai/evidently/blob/v0.4.39/setup.py#L66
```suggestion
    - uvicorn >=0.22.0
    - colorama >=0.4 # [win]
    - httptools >=0.5.0
    - python-dotenv >=0.13
    - uvloop >=0.14.0,!=0.15.0,!=0.15.1 # [unix]
    - watchfiles >=0.13
    - websockets >=10.4
```
https://inspector.pypi.io/project/uvicorn/0.30.6/packages/5a/01/5e637e7aa9dd031be5376b9fb749ec20b86f5a5b6a49b87fabd374d5fa9f/uvicorn-0.30.6.tar.gz/uvicorn-0.30.6/pyproject.toml#line.39

---

didn't catch that!

---

there's pyyaml too